src/main/java/com/sprint/mission/discodeit/controller/ReceiveMessageController.java

```diff
@@ -0,0 +1,66 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.channel_service_dto.ChannelResponseDto;
+import com.sprint.mission.discodeit.dto.channel_service_dto.EnterChannelRequest;
+import com.sprint.mission.discodeit.dto.readstatus_dto.ReadStatusResponseDto;
+import com.sprint.mission.discodeit.dto.user_service_dto.UserResponseDto;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.service.MessageService;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import com.sprint.mission.discodeit.service.UserService;
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.List;
+
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/discodeit/receive-message")
```

Comment:
```suggestion
@RequestMapping("/v1/discodeit/messages:receive")
```
이런식의 표현이 훨씬 RESTFul 합니다.